libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
```

Comment:
Let's bite the bullet... 
I would move it to Text instead of XML (after all it's more a text issue than a XML one) and then deal with all the tags from https://github.com/musescore/MuseScore/blob/master/libmscore/text.cpp#L1137